Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -2079,11 +2093,227 @@ static ErrorOr<void> read_lf_coefficients(LittleEndianInputBitStream&, FrameHead
     return Error::from_string_literal("JPEGXLLoader: Implement reading LF coefficients");
 }
 
+// I.1 - Transform types
+enum class TransformType : u8 {
+    DCT8x8 = 0,
+    Hornuss = 1,
+    DCT2x2 = 2,
+    DCT4x4 = 3,
+    DCT16x16 = 4,
+    DCT32x32 = 5,
+    DCT16x8 = 6,
+    DCT8x16 = 7,
+    DCT32x8 = 8,
+    DCT8x32 = 9,
+    DCT32x16 = 10,
+    DCT16x32 = 11,
+    DCT4x8 = 12,
+    DCT8x4 = 13,
+    AFV0 = 14,
+    AFV1 = 15,
+    AFV2 = 16,
+    AFV3 = 17,
+    DCT64x64 = 18,
+    DCT64x32 = 19,
+    DCT32x64 = 20,
+    DCT128x128 = 21,
+    DCT128x64 = 22,
+    DCT64x128 = 23,
+    DCT256x256 = 24,
+    DCT256x128 = 25,
+    DCT128x256 = 26,
+};
+
+// NOTE: In the spec, DCT matrices use "matrices order" so DCT16x8 is actually
+//       16 rows and 8 columns. This function return the size in "image order"
+//       with columns first and rows in second.
```

Comment:
It also seems to return number of 8 pixel blocks, while the enums contain pixel dimensions. Maybe worth mentioning as well?